domains/_github-pages-challenge-superuser-l.this.is-cool.dev.json

```diff
@@ -6,11 +6,18 @@
 
   "owner": {
     "repo": "https://github.com/tr1xem",
+
     "email": "trix@lawliet.tk"
   },
 
   "record": {
-    "TXT": ["e388731d5ed45dd2aefcd0338c0a99"]
+    "CNAME":"superuser-l.github.io",
```

Comment:
Cannot have a CNAME record in conjunction with other records and vice versa.

Reply:
but github needs it